code/service/src/main/scala/com/namely/chiefofstate/readside/ReadSideManager.scala

```diff
@@ -29,7 +30,7 @@ class ReadSideManager(
     interceptors: Seq[ClientInterceptor],
     dbConfig: ReadSideManager.DbConfig,
     readSideConfigs: Seq[ReadSideConfig],
-    numShards: Int) {
+    numShards: Int)(implicit ec: ExecutionContext) {
```

Comment:
I think it would be better to make this arg explicit here and set it as `implicit` closer to where you need that